package.json

```diff
@@ -6,8 +6,8 @@
   "main": "dist/index.js",
   "type": "module",
   "scripts": {
+    "build": "esbuild src/main.ts --bundle --platform=node --target=node16 --outfile=dist/index.js && shx cp node_modules/eipw-lint-js/eipw_lint_js_bg.wasm dist && echo { \"type\": \"commonjs\" } > dist/package.json",
```

Comment:
Can we keep the compiled version an ES module?

Reply:
It's a lot easier when it's commonjs, as node has better support for it.

Reply:
I guess as long as it works, I don't care much.